### DISCUSSION TITLE
embeddable form on website [Oct 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6719] [Discord Threads]
> 🙋‍♂️ **halphalp**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### 1. I think you need to set `scrolling="yes"` in your `<iframe>` tag   
#### 2. There is no way to do that.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1150014830598705232)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
